charts/osm/templates/osm-configmap.yaml

```diff
@@ -8,7 +8,7 @@ data:
   egress: {{ .Values.OpenServiceMesh.enableEgress | quote }}
   envoy_log_level: {{ .Values.OpenServiceMesh.envoyLogLevel | quote }}
   enable_debug_server: {{ .Values.OpenServiceMesh.enableDebugServer | quote }}
-  prometheus_scraping: "true"
+  prometheus_scraping: {{ .Values.OpenServiceMesh.enablePrometheus | quote }}
```

Comment:
The `.Values.OpenServiceMesh.enablePrometheus` is used to deploy Prometheus resources in osm but this field needs to be set true for BYO Prometheus, so I am not 100% sure if we should link the two.

Reply:
I see. I can make a separate flag?

Reply:
yes, a separate flag would help making it configurable for the demo vs a regular install